wpilibc/src/main/native/cpp/PneumaticsControlModule.cpp

```diff
@@ -160,3 +163,29 @@ void PneumaticsControlModule::SetOneShotDuration(int index,
 bool PneumaticsControlModule::CheckSolenoidChannel(int channel) const {
   return HAL_CheckCTREPCMSolenoidChannel(channel);
 }
+
+int PneumaticsControlModule::CheckAndReserveSolenoids(int mask) {
+  std::scoped_lock lock{m_reservedLock};
+  uint32_t uMask = static_cast<uint32_t>(mask);
```

Comment:
Is there a reason the param/return value aren’t uint32_t?

Reply:
Since we pretty much standardized on int everywhere at the wpilib layer.